### DISCUSSION TITLE
Make username simpler : )

### DISCUSSION THREAD

**Discussion by steveiliop56** [lllllllillllllillll/DweebUI#46] [General]
It would be easier for you to have a simpler username so we can easily download the image and not having to find lllllllllllillllllllilllll.

**Comment by lllllllillllllillll** [comment]
I don't know what you're talking about, that's just the name my parents gave me.

I'm considering a new username and possibly a different name for the project.
Currently rewriting a lot of the code.

**Comment by steveiliop56** [comment]
No keep the project name dweebui is a cool name
